packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/listeners/listener/l7/routing.js

```diff
@@ -0,0 +1,75 @@
+import { TRACKING_SUFFIX } from './constants';
+import { TRACKING_SUFFIX as LISTENERS_TACKING_SUFFIX } from '../../constants';
+import { LISTENER_PROTOCOLS_ENABLING_POLICIES } from '../../list/constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'octavia-load-balancer.loadbalancer.listeners.listener.l7Policies',
+    {
+      url: '/l7',
+      views: {
+        'octaviaLoadBalancerView@octavia-load-balancer':
+          'octaviaLoadBalancerL7',
+      },
+      resolve: {
+        breadcrumb: () => 'L7 Policies',
+        listener: /* @ngInject */ (
+          OctaviaLoadBalancerListenersService,
+          projectId,
+          region,
+          listenerId,
+        ) =>
+          OctaviaLoadBalancerListenersService.getListener(
+            projectId,
+            region,
+            listenerId,
+          ),
+        goToListenerListingPage: /* @ngInject */ ($state) => () => {
+          $state.go('octavia-load-balancer.loadbalancer.listeners.list');
+        },
+        trackL7Base: /* @ngInject */ (trackRoot) =>
+          `${trackRoot}::${LISTENERS_TACKING_SUFFIX}::${TRACKING_SUFFIX}`,
+        trackL7Action: /* @ngInject */ (atInternet, trackL7Base) => (hit) =>
+          atInternet.trackClick({
+            name: `${trackL7Base}::${hit}`,
+            type: 'action',
+          }),
+        trackL7Page: /* @ngInject */ (atInternet, trackL7Base) => (hit) =>
```

Comment:
It is not possible to use atInternet property on routing for track page ?

Reply:
As seen together, this is not meant to be use as the tracking for this state but rather to offer a tracking function for children state (allowing them to avoid importing tracking constants from parent folder)